src/js/components/DataChart/stories/Axis.js

```diff
@@ -24,17 +24,20 @@ for (let i = 0; i < 7; i += 1) {
   });
 }
 
-const AxesDataChart = () => (
+const Example = () => (
   <Grommet theme={grommet}>
     <Box align="center" justify="start" pad="large">
       <DataChart
         data={data}
-        chart={{ key: 'percent', type: 'bar' }}
-        xAxis={{ key: 'day', guide: true }}
-        yAxis={{ guide: true, labels: 3, suffix: '%' }}
+        series={['day', { property: 'percent', suffix: '%' }]}
+        chart="percent"
```

Comment:
It is not clear what it means to have a string as the value of `chart`.  
The `chart` prop is documented to have only chart type or an array of it, we should add what it means to use a string and support it on TS as well.

Reply:
Enhanced documentation